packages/docgen/src/test/get-intermediate-representation.js

```diff
@@ -50,37 +50,6 @@ describe( 'Intermediate Representation', () => {
 		} );
 	} );
 
-	it( 'type parse error handling', () => {
```

Comment:
I guess this makes sense to remove: as I understand it, `comment-parser` does actually not parse the type, just passes it through.

Reply:
I did add a new error case just now for when the parsing fails to parse it as a JSDoc comment (in the case of malformed comment, for example). I don't know if it's possible IRL but was easy to test for and caught an null pointer exception that was possible in that case.